src/HospitalAPI/Emails/SendgridProvider.cs

```diff
@@ -14,6 +14,7 @@ public class SendgridProvider: IEmailService
 
         public SendgridProvider()
         {
+            Console.WriteLine(Environment.GetEnvironmentVariable("SENDGRID_API_KEY"));
             _sendgrid = new SendGridClient(Environment.GetEnvironmentVariable("SENDGRID_API_KEY"));
```

Comment:
@srdjan1210 is this something connected to email? @kkrizan75 didnt edit this file